Jenkinsfile

```diff
@@ -35,10 +45,8 @@ node ("openshift-test-pipeline-slave") {
         stage("Validate OpenShift user creation") {
             sh("""
                 RANDOM_PASSWORD=\$(openssl rand -base64 20 | cut -d= -f1);
-                ssh -o StrictHostKeyChecking=no -i ssh_key cloud-user@\$Bastionip \
-                    "cd /usr/share/ansible/openshift-deployment-ansible/tools/ ; ./create-user.sh testUser \$RANDOM_PASSWORD"
-                ssh -o StrictHostKeyChecking=no -i ssh_key cloud-user@\$Bastionip \
-                    "oc login https://ocp.\$Domainsuffix:8443 --insecure-skip-tls-verify=true -u testUser -p \$RANDOM_PASSWORD"
+                ssh -o StrictHostKeyChecking=no -i ssh_key cloud-user@\$Bastionip "cd /usr/share/ansible/openshift-deployment-ansible/tools/ ; ./create-user.sh testUser \$RANDOM_PASSWORD"
```

Comment:
Please can you change the casing of the Jenkins variable to better match: https://github.com/UKCloud/openshift-deployment-jenkins-pipeline/blob/3dc2e1b1f4c15dfc1166fc3e1ff4ab805b42c3f6/jenkins-pipelines/openshift/Jenkinsfile#L209 ?